.github/workflows/release-10_docker-manual.yml

```diff
@@ -13,25 +13,34 @@ on:
         required: true
       prerelease:
         description: is prerelease
-        default: false
+        default: "false"
```

Comment:
Just curious why the double quotes needed here?

Reply:
I was curious as well :)
`true` is a boolean and considered as such unless you quote it. So using `false` will not fail but GH will NOT load the default and the user is left with the work. Using `"false"` however makes GH happy as it takes the **string** `false` and uses it as default.

Reply:
So without quoting, you get:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/738724/176650606-47d2846a-7170-43bc-bed6-c4bbf23aaf2b.png">

instead of:
<img width="349" alt="image" src="https://user-images.githubusercontent.com/738724/176650674-1a2ded30-e149-4238-b0e9-71a919113999.png">